src/components/forms/dataset/DatasetCardForSelect.vue

```diff
@@ -26,12 +27,12 @@ const props = defineProps({
   }
 })
 
-const ownerName = useOwnerName(props.dataset)
+const ownerName = useOwnerName(props.dataset as Owned)
```

Comment:
Toujours dangereux les `as`... Pourtant le `DatasetV2` devrait hériter de `Owned`?

Reply:
C'est un peu le bazarre parce que `useOwnerName` peut prendre n'importe quel objet (bouquet, dataset, etc.)
En théorie ils sont tous censés hérité du `Owned` mais la manière dont il est construit n'a pas l'air de plaire à TS.

```ts
useOwnerName(owned: MaybeRefOrGetter<Owned | null>)
```

`type 'DatasetV2' is not assignable to parameter of type 'MaybeRefOrGetter<Owned | null>'`

Reply:
C'est étonnant, d'autant que ça marche avec notre type `Topic` dans `BouquetCard`. Je rajouterais un `FIXME:`, idéalement avec un lien vers une issue à ouvrir upstream sur data.gouv.fr.

Reply:
Si je créé un type local et que j'explicite le `Owned` ça résout le problème 😅 
```ts
import type { DatasetV2, Owned } from '@datagouv/components/ts'
export type DatasetV2Fix = DatasetV2 & Owned
```

Reply:
Ah! Alors peut-être définir ce type avec un FIXME :-)

Reply:
fixed by #551